docs/plugins/woodpecker-plugins/plugins.json

```
@@ -35,6 +35,11 @@
       "docs": "https://codeberg.org/qwerty287/woodpecker-regex-check/raw/branch/main/docs.md",
       "verified": false
     },
+    {
+      "name": "Gitea Pull comment",
+      "docs": "https://github.com/markopolo123/gitea-comment-plugin/blob/main/docs.md",
```

```suggestion
      "docs": "https://raw.githubusercontent.com/markopolo123/gitea-comment-plugin/main/docs.md",
```
It has to be the raw file.